Bottom: cecf877414f949e1f63ac3d8ff08d60052cd17d3
Top:    164b324260b08088b87f4df598bbecaea49579dc
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-08-09 09:06:00 -0500

Refresh of dump-host-vars

---
diff --git a/ansible-playbook/site.yml b/ansible-playbook/site.yml
index cfc2595a..769c01fe 100644
--- a/ansible-playbook/site.yml
+++ b/ansible-playbook/site.yml
@@ -5,6 +5,7 @@
   hosts: all
   tags:
     - debugging
+    - always
   tasks:
   - template:
       src: templates/dump_variables
diff --git a/ansible-playbook/templates/dump_variables b/ansible-playbook/templates/dump_variables
index 6a8b42ce..97fd2e34 100644
--- a/ansible-playbook/templates/dump_variables
+++ b/ansible-playbook/templates/dump_variables
@@ -1,3 +1,3 @@
 HOSTVARS (ANSIBLE GATHERED, group_vars, host_vars) :
 
-{{ hostvars[inventory_hostname] | to_yaml }}
+{{ hostvars | to_yaml }}
